### PR TITLE
chore(apps/interface): refactor ArrowUpIcon

### PR DESCRIPTION
**Pull request by math-marcellino**
## Scope
List of affected projects:

- apps/interface

## Description
We need to refactor the `ArrowUpIcon` component.

Return `Icon` directly instead of using `return` keyword.

Update this:

```
const ArrowUpIcon = (props: IconProps) => {
	return <Icon />
}
```

to this:

```
const ArrowUpIcon = (props: IconProps) => (
	<Icon />
)
```

## Action items
Action items for this pull request:

- [x] Refactor the return function

## Checklist
You should check this before requesting for review:

- [x] Branch name use the the following format RIS-{NUMBER}
- [x] Pull request title is "chore(apps/interface): refactor ArrowUpIcon"
- [x] Make sure CSS styling is same as the spec (padding, border etc)
- [x] Make sure new files are 100% covered in [Risedle Code Coverage](https://coverage.risedle.com)

### CODE REVIEW

File: apps/interface/components/Icons/ArrowUp.tsx [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
+    <Icon data-testid="ArrowUpIcon" viewBox="0 0 16 16" {...props}>
+        <path
+            fillRule="evenodd"
+            clipRule="evenodd"
+            d="M7.81246 2.14645C8.00772 1.95118 8.32431 1.95118 8.51957 2.14645L12.5196 6.14645C12.7148 6.34171 12.7148 6.65829 12.5196 6.85355C12.3243 7.04882 12.0077 7.04882 11.8125 6.85355L8.66602 3.70711L8.66602 12.5C8.66602 12.7761 8.44216 13 8.16602 13C7.88987 13 7.66602 12.7761 7.66602 12.5L7.66602 3.70711L4.51957 6.85355C4.32431 7.04882 4.00772 7.04882 3.81246 6.85355C3.6172 6.65829 3.6172 6.34171 3.81246 6.14645L7.81246 2.14645Z"
+            fill="currentColor"
+        />
+    </Icon>
+);
 export default ArrowUpIcon;
```
Comment (pyk) [inline apps/interface/components/Icons/ArrowUp.tsx:13]: @math-marcellino please add new line above this line

**Review by pyk — CHANGES_REQUESTED** [review]

File: apps/interface/components/Icons/ArrowUp.tsx (hunk 1 above)
Comment (math-marcellino) [reply apps/interface/components/Icons/ArrowUp.tsx:13]: gotcha, sorry I missed in this one too sir

Comment (math-marcellino) [reply apps/interface/components/Icons/ArrowUp.tsx:14]: fixed sir @pyk

**Review by pyk — APPROVED** [review]
Thanks @math-marcellino !

**Pull request merged**